Representable/representable
# add census block mapping for all states

**changes**
- adds ability to switch units from block groups -> blocks for every state
- two states (illinois + oklahoma) use 2010 blocks, with population data included, all the rest use 2020 blocks and have no population
- add population button in lower right corner of mapping page (if population is available)
- adds "want different units?" button with accompanying modal explaining difference between units

**to test**
- python manage.py runserver
- create a community in any state
- click "want different units?" button on mapping page and switch to census blocks
- check you can submit a community
- check that you can submit a community comprised of block groups
- check that the population only appears in the lower right corner of the mapping page when expected (blocks + illinois/oklahoma, blockgroups + all states except nc, az, mi, nv, nj, nm, pa, ut, va, wa)

**screenshots**
illinois block group mapping
![image](https://user-images.githubusercontent.com/41943646/114930472-0b809a00-9e03-11eb-8a75-415546e5281a.png)
modal for switching
![image](https://user-images.githubusercontent.com/41943646/114930491-13d8d500-9e03-11eb-9c3b-acd5a28b3986.png)
illinois block mapping
![image](https://user-images.githubusercontent.com/41943646/114930585-2c48ef80-9e03-11eb-8191-9dfec1f32bb2.png)
